golang/internal/grpc/grpc.go

```diff
@@ -439,7 +437,7 @@ func executeSecretList(
 		Keys:      keys,
 	}
 
-	_, err = grpcConn.Client.SecretsList(ctx, resp)
+	_, err = grpcConn.Client.GetSecretList(ctx, resp)
```

Comment:
Why is this renamed to `GetSecretList`? It was called `SecretList` because the agent sends data to crux, so it's not 'getting' anything.

Reply:
Bad merge